spec/support/sdam_monitoring/standalone.yml

```diff
@@ -0,0 +1,74 @@
+description: "Monitoring a standalone connection"
+uri: "mongodb://a:27017"
+phases:
+  -
+    responses:
+      "a:27017":
+        ok: 1
+        ismaster: true
+        minWireVersion: 0
+        maxWireVersion: 4
+    outcome:
+      events:
+        -
+          topology_opening_event:
+            topologyId: "42"
+        -
+          server_opening_event:
+            topologyId: "42"
+            address: "a:27017"
+        -
+          server_description_changed_event:
+            topologyId: "42"
+            address: "a:27017"
+            previousDescription:
+              address: "127.0.0.1:27017"
```

Comment:
I thought I had read in the SDAM spec as well about the default for the server description but reading it again I don't see it... So maybe leave it out? I think some implementations will default with an "empty" description and some might default with the address provided and type unknown.